src/compressed_tensors/quantization/quant_scheme.py

```diff
@@ -57,12 +57,9 @@ def default_scheme(
             # default to quantizing all Linear layers
             targets = ["Linear"]
 
-        # default to 8 bit integer symmetric quantization
-        # for weights
-        weights = QuantizationArgs(num_bits=8, symmetric=True)
+        weights = None
 
-        # default to 8 bit integer asymmetric quantization
-        input_activations = QuantizationArgs(num_bits=8, symmetric=True)
+        input_activations = None
 
         # Do not quantize the output activations
         # by default
```

Comment:
nit: now that we aren't quantizing anything by default can we update the comment on line 64-65 and just set everything at the same time, ie

```
# by default, activations and weights are left unquantized
weights = None
input_activations = None
output_activations = None
```